controllers/bounce_processes.go

```diff
@@ -255,9 +255,9 @@ func getAddressesForUpgrade(logger logr.Logger, r *FoundationDBClusterReconciler
 		return nil, &requeue{curError: err}
 	}
 
```

Comment:
A comment about why we don't want to do fault tolerance check here would be nice.

Reply:
Good point, I'm adding the comment.